v1/recommend-description.yaml

```diff
@@ -0,0 +1,155 @@
+tags:
+  - name: Recommendation
+    description: contribution recommendations
+paths:
+  /recommendation/description/addition/{target}:
+    get:
+      tags:
+        - Recommendation
+      summary: Recommend Wikibase items missing a description in the target language
+      parameters:
+        - name: target
+          in: path
+          description: The target wiki language code for suggestions
+          required: true
+          schema:
+            type: string
+      responses:
+        200:
+          description: list of candidates for description addition
+          content:
+            application/json:
+              schema:
+                $ref: '#/components/schemas/description_recommendation_result'
+        default:
+          description: Error
+          content:
+            application/problem+json:
+              schema:
+                $ref: '#/components/schemas/problem'
+      x-request-handler:
+        - get_from_backend:
+            request:
+              uri: '{{options.host}}/{{domain}}/v1/description/addition/{target}'
+      x-amples:
+        - title: Description addition suggestions
+          request:
+            params:
+              domain: www.wikidata.org
+              target: zh
+            response:
+              status: 200
+              headers:
+                content-type: application/json
+              body:
+                - pageid: /.+/
+                  ns: 0
+                  title: /.+/
+                  wikibase_item:
+                    type: item
+                    id: /.+/
+                    labels: /.+/
+                    descriptions: /.+/
+                    sitelinks: /.+/
+
+  /recommendation/description/translation/from/{source}/to/{target}:
+    get:
+      tags:
+        - Recommendation
+      summary: Recommend Wikibase items with a description in the source language but missing one in the target language
+      parameters:
+        - name: source
+          in: path
+          description: The source wiki language code for suggestions
+          required: true
+          schema:
+            type: string
+        - name: target
+          in: path
+          description: The target wiki language code for suggestions
+          required: true
+          schema:
+            type: string
+      responses:
+        200:
+          description: list of candidates for description translation
+          content:
+            application/json:
+              schema:
+                $ref: '#/components/schemas/description_recommendation_result'
+        default:
+          description: Error
+          content:
+            application/problem+json:
+              schema:
+                $ref: '#/components/schemas/problem'
+      x-request-handler:
+        - get_from_backend:
+            request:
+              uri: '{{options.host}}/{{domain}}/v1/description/translation/from/{source}/to/{target}'
+      x-amples:
```

Comment:
Needs x-monitor: true as well.

Actually. while you're at it, could you amend our tests to fail if there IS x-amples and no x-monitoring. Right about here: https://github.com/wikimedia/restbase/blob/master/test/features/specification/monitoring.js#L43

Reply:
`x-monitor` is actually not needed if it's value it's supposed to be `true`. The default is `x-monitor: true` in service-checker

Reply:
olala... that's the shitty part of having the checker reimplemented in node for testing. out tests need x-monitor

Reply:
Which config file should we now be using for ordinary testing?  These now pass with `x-monitor: true` when using `config.fullstack.test.yaml` but only for beta cluster domains.  But won't this put beta cluster domains on the Swagger doc page, even in production?

Also, I noticed that most endpoints have `x-monitor: false` in RESTBase.  That's why I did likewise in the new PR before seeing this comment.